### DISCUSSION TITLE
Extract ESPN Fantasy Basketball too?

### DISCUSSION THREAD

**Discussion by lightsout7** [adamvaldez/ESPN_Extractor#6] [Ideas]
I was curious if this tool could be refitted to extract statistics for ESPN fantasy basketball as well.

**Comment by adamvaldez** [comment]
Yes, this tool is using the [espn_api](https://github.com/cwendt94/espn-api) and that supports football, basketball, baseball, and hockey.

I can create a ticket for this, not sure when I or another developer working on the project will be able to get to it.

**Reply by adamvaldez** [reply]
@lightsout7 this request is no trackable in issues https://github.com/adamvaldez/ESPN_Extractor/issues/7
